Sources/StreamChatSwiftUI/ChatChannel/Reactions/ReactionsUsersView.swift

```diff
@@ -70,14 +72,60 @@ struct ReactionsUsersView: View {
             .background(Color(colors.background))
             .cornerRadius(16)
 
-            if !message.isRightAligned {
+            if !viewModel.isRightAligned {
                 Spacer()
             }
         }
         .accessibilityIdentifier("ReactionsUsersView")
     }
 }
 
+class ReactionsUsersViewModel: ObservableObject, ChatMessageControllerDelegate {
+    @Published var reactions: [ChatMessageReaction] = []
```

Comment:
maybe show the first batch from the latest reactions from the message?

Reply:
since like this we're always making an API call to start pagination, right?

Reply:
No, we are not because the initail data will be already from the latest reactions. So in case there is no more pages, we don't perform the request